yammer-extra/findbugs.exclude.xml

```diff
@@ -0,0 +1,44 @@
+<!--
+  ~ Copyright 2015 Groupon.com
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<FindBugsFilter>
+
+     <!-- Match all DLS_DEAD_LOCAL_STORE violations. As per FindBugs
+         documentation these generate numerous false positives under Sun/Oracle
+         compilers.
+
+         http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE
+     -->
+     <Match>
+         <Bug pattern="DLS_DEAD_LOCAL_STORE" />
+     </Match>
+
+     <!-- Match all RV_RETURN_VALUE_IGNORED_BAD_PRACTICE violations on all unit
+        test files since Mockito usage can cause this violation when stating
+        expectations.
+     -->
+     <Match>
+         <Or>
+            <Bug pattern="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE" />
```

Comment:
You should find a way to use this from the parent pom and find a way to merge them ( :  As of right now, it hasn't come up.  I'm happy to add this on an as-needed basis.